source/MaterialXContrib/MaterialXNode/resources/standard_surface_default.xml

```diff
@@ -531,7 +533,7 @@ vec3 mx_environment_irradiance(vec3 N)
 //
 vec2 mx_get_target_uv(vec2 uv)
 {
-   return uv;
+    return vec2(uv.x, 1.0 - uv.y);
```

Comment:
I generated this code without the V-flip added in so just manually fixing. The C++ code does the correct thing now.